api/devices/schema.py

```diff
@@ -78,8 +78,6 @@ def mutate(self, info, device_id, **kwargs):
         ).first()
         if not exact_device:
             raise GraphQLError("DeviceId not found")
```

Comment:
Kindly add a space between the Device and ID message. For aesthetic purposes

Reply:
I'll fix that.👍🏼